librato/resource_librato_alert.go

```diff
@@ -334,7 +334,8 @@ func resourceLibratoAlertConditionsGather(d *schema.ResourceData, conditions []l
 func resourceLibratoAlertAttributesGather(d *schema.ResourceData, attributes *librato.AlertAttributes) []map[string]interface{} {
 	result := make([]map[string]interface{}, 0, 1)
 
-	if attributes != nil {
+	// Treat an empty hash of attributes as being identical to no attributes set at all.
+	if attributes != nil && attributes.RunbookURL != nil {
 		retAttributes := make(map[string]interface{})
 		if attributes.RunbookURL != nil {
```

Comment:
Nit: since this is now checked on 338, could the contents of the if block be reduced to this?

```go
result = append(result, map[string]interface{
  "runbook_url": *attributes.RunbookURL,
})

Reply:
Good catch @bernerdschaefer, thanks! Looks correct to me. I'll verify and make the change assuming all is good.